src/styles/Blocks/cards.scss

```diff
@@ -0,0 +1,22 @@
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  width: 944px;
```

Comment:
I don't know even if you need specified width, your card elements dictate the width already. But if you do need for some reason, i don't think it should be sth like 944px. Maybe you can leave some comment to code so that reviewer can understand.

Reply:
support Efdal
code itself can be self-explanatory, say if used variables:
 `width: (4 * $card-width) + (3 * $card-horizonal-spacing);`
would perfectly explain itself